### ISSUE TITLE
Debian 11 (Bullseye): `intellij-idea-community`: Cannot install: Depends: `lib32gcc1` but it is not installable

### ISSUE THREAD

**Issue by sbrunov** [JonasGroeger/jetbrains-ppa#33]
Hi, everyone!

# Context
## Operating system
Using Debian 11 (Bullseye) `testing` with some packages from `unstable` and `experimental`.
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux bullseye/sid
Release:	testing
Codename:	bullseye
```

# Problem
Have just faced the installation-related problem with the `intellij-idea-community` package.

Cannot install it because of the following error:
```
# apt install intellij-idea-community 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 intellij-idea-community : Depends: lib32gcc1 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

According to the [Debian -- Details of package lib32gcc1 in buster](https://packages.debian.org/buster/lib32gcc1) page, It seems that the package is not available for Debian 11 (Bullseye): please, see the available Debian versions in the right top corner of the page.

# Some thoughts on possible solution
Does the `intellij-idea-community` package **actually** require the `lib32gcc1` package?  If it does not, then it is worth removing the `lib32gcc1` package dependency.

Best regards,  
Sergey Brunov.

**Comment by jdeluyck** [comment]
lib32gcc1 seems to have been dropped recently, and gcc-base-10 upgraded:

  * Stop building the libgcc compat packages for bullseye/sid.

**Comment by sbrunov** [comment]
Dear Jonas Gröger (@JonasGroeger),

Could you please have a look at the problem and share your thoughts on it?

Best regards,  
Sergey Brunov.

**Comment by jdeluyck** [comment]
As @JonasGroeger is not reacting, I've replicated the travis ci setup, and i'm building the packages and serving them. Once Jonas comes back I'm more than willing to hand this back to him. 

https://github.com/jdeluyck/jetbrains-ppa
https://ppa-jetbrains.kcore.org/file/ppa-jetbrains/index.html

**Comment by JonasGroeger** [comment]
Fixed with #34
